base/migrations/0016_alter_intranetindexpage_editor_and_more.py

```diff
@@ -0,0 +1,145 @@
+# Generated by Django 4.1.10 on 2023-07-26 16:33
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("staff", "0028_alter_staffpage_bio"),
+        ("base", "0015_localmedia"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="intranetindexpage",
+            name="editor",
+            field=models.ForeignKey(
+                null=True,
+                on_delete=django.db.models.deletion.SET_NULL,
+                related_name="%(app_label)s_%(class)s_editor",
+                to="staff.staffpage",
+            ),
+        ),
+        migrations.AlterField(
+            model_name="intranetindexpage",
+            name="page_maintainer",
+            field=models.ForeignKey(
+                null=True,
+                on_delete=django.db.models.deletion.SET_NULL,
+                related_name="%(app_label)s_%(class)s_maintainer",
+                to="staff.staffpage",
+            ),
+        ),
+        migrations.AlterField(
+            model_name="intranetplainpage",
+            name="editor",
+            field=models.ForeignKey(
+                null=True,
+                on_delete=django.db.models.deletion.SET_NULL,
+                related_name="%(app_label)s_%(class)s_editor",
+                to="staff.staffpage",
+            ),
+        ),
+        migrations.AlterField(
+            model_name="intranetplainpage",
+            name="page_maintainer",
+            field=models.ForeignKey(
+                null=True,
+                on_delete=django.db.models.deletion.SET_NULL,
+                related_name="%(app_label)s_%(class)s_maintainer",
+                to="staff.staffpage",
+            ),
+        ),
+        migrations.AlterField(
+            model_name="localmedia",
+            name="caption_language",
+            field=models.CharField(
+                choices=[
+                    ("af", "Afrikaans"),
```

Comment:
Huh, interesting to see our Wagtail site already has a dictionary mapping ISO language codes to language names.  Did not know that.

Reply:
Yes, there is some of that. I think there's a Django module you can just import and use as well. I'm fuzzy on the details but I recall something like that.